### PR TITLE
Fix bug trying to deserialize jsonschema validation error

### PR DESCRIPTION
**Pull request by lferran**
This is happening for fields with `schema.JSONField` type, if a type that doesn't match the schema is sent.

### CODE REVIEW

File: guillotina/exceptions.py [hunk 1]
```diff
@@ -216,7 +216,15 @@ def json_err_list(self, errors):
...
                 if hasattr(exc, "errors") and exc.errors:
-                    error["errors"] = self.json_err_list([e.json() for e in exc.errors])
+                    inner_errors = []
+                    for e in exc.errors:
+                        try:
+                            inner_errors.append(e.json())
+                        except AttributeError:
+                            # jsonschema.exceptions.ValidationError
+                            # does not have json() method
+                            inner_errors.append({"error_str": str(e)})
```
Comment (lferran) [inline guillotina/exceptions.py:226]: not sure what to do here. 
 
I could also do:
```python
if isinstance(jsonschema.AttributeError, e):
   inner_errors.append({"message": e.message})
else:
   inner_errors.append(e.json())
```

What's your opinion?

Comment (masipcat) [reply guillotina/exceptions.py:226]: I like this option:

```
if isinstance(jsonschema.exceptions.ValidationError, e):
   inner_errors.append({"message": e.message})
else:
   inner_errors.append(e.json())
```

Comment (masipcat) [reply guillotina/exceptions.py:226]: @lferran is this pr still a draft? is ready for review? I'd like to have this fix :)

**Review by masipcat — APPROVED** [review]
LGTM!

**Review by masipcat — APPROVED** [review]

**Review by bloodbare — APPROVED** [review]

**Pull request merged**